tests/pfcwd/test_pfcwd_all_port_storm.py

```diff
@@ -174,6 +174,23 @@ def set_storm_params(duthost, fanout_graph, fanouthosts, peer_params):
     return storm_hndle
 
 
+def resolve_arp(duthost, ptfhost, test_ports_info):
+    """
+    Populate ARP info for the DUT vlan port
+
+    Args:
+        ptfhost: ptf host instance
+        test_ports_info: test ports information
+    """
+    for port, port_info in test_ports_info.items():
+        if port_info['test_port_type'] == 'vlan':
```

Comment:
I checked the test history on T0, it's consistently passing. May I know when this ARP populate is needed?

Reply:
@echuawu Can you please address this comment?